.goreleaser.yaml

```diff
@@ -1,12 +1,9 @@
-partial:
-  by: target
```

Comment:
Why this change? This is here to avoid running into disk space issues

Reply:
My bad. Was deleted by accident

Reply:
np!